pyproject.toml

```
@@ -1,104 +1,76 @@
-[tool.poetry]
+[project]
 name = "dhlab"
 version = "2.41.0"
 description = "Text and image analysis of the digital collection (books, newspapers, periodicals, and images) at the National Library of Norway"
-authors = ["The Digital Humanities Lab at The National Library of Norway (NB) <dh-lab@nb.no>"]
-license = "MIT"
+authors = [
+    {name = "The Digital Humanities Lab at The National Library of Norway", email = "dh-lab@nb.no"}
+]
+license = {text = "MIT"}
 readme = "README.md"
-homepage = "https://www.nb.no/dh-lab/"
-documentation = "https://dhlab.readthedocs.io/en/stable/"
-repository = "https://github.com/NationalLibraryOfNorway/DHLAB"
+requires-python = "<3.13,>=3.10"
```

```suggestion
requires-python = ">=3.10"
```
I don't believe we need the pin on the upper version number with PDM or uv?

---

We do - the library breaks with numpy>=2 and scipy>=1.15, because of the legacy code that is imported at the top level __init__.py.  When I tried to  restrict only numpy and scipyv versions, pdm complained that the python version range was incompatible with the dependencies.